src/main/tscfg/example/duration2.spec.conf

```diff
@@ -0,0 +1,23 @@
+// GenOpts: --durations
+durations {
+  # optional duration; reported Long (Option[Long] in scala) is null (None) if value is missing
```

Comment:
Shouldn't it say "... reported **java.time.Duration** (Option[**java.time.Duration**] in scala) ..."?